### ISSUE TITLE
[10.0] [11.0][EN] Opening a New Tab from an Odoo's menu, removes all menus on the new tab.

### ISSUE THREAD

**Issue by gustavovalverde** [odoo/odoo#22379]
#### Impacted versions:
10.0, 11.0, maybe 9.0

#### Steps to reproduce:
Using Chrome, right click on a menu, open a new tab from it
![image](https://user-images.githubusercontent.com/16890942/35150752-ef2caae8-fcf1-11e7-8fcd-30b004b51980.png)

#### Current behavior:
The opened tab, most of the time hides all menus, making it necessary to always refresh
![image](https://user-images.githubusercontent.com/16890942/35150692-bbffff76-fcf1-11e7-9cee-3a525e11b59d.png)

#### Expected behavior:
New tab shows the menus

**Comment by gustavovalverde** [comment]
@Yenthe666 ping, as I don't know who deals with UI

**Comment by Yenthe666** [comment]
Ah yes, this one, I've seen it many times and I personally hate it too.
I believe @adr-odoo falls under this one too. (CC @mart-e)

**Comment by sswapnesh** [comment]
It also removes breadcrumb on Refresh
![1](https://user-images.githubusercontent.com/16624719/42586079-da8da5ba-8554-11e8-8199-9077f4764b13.png)

Refresh 

![2](https://user-images.githubusercontent.com/16624719/42586101-ea0f2d88-8554-11e8-8806-d450515723be.png)

**Comment by gustavovalverde** [comment]
With all the changes going on with BS4 and the frontend (in master), this would be a great time to solve this one cc: @mart-e

**Comment by mart-e** [comment]
Actually we already discussed about it (can't find the issue back).
As long as you have the action and menu_id in the URL, you should be "refresh proof". If you cross records through links (many2one, stat buttons,..) no way to recover it (it would imply storing the full path in the URL which is a bit overkill).

There may have an issue with the menu_id in the link that is wrong when you try to open in a new tab.
@ged-odoo I think we already discussed about it.

**Comment by pedrobaeza** [comment]
@mart-e have you considered any time a recoverable breadcrumb history through URL (with some kind of short refs like IDs)?

**Comment by mart-e** [comment]
@pedrobaeza no, I have to admit I find it quite difficult in comparison to the benefit it brings

**Comment by pedrobaeza** [comment]
Well, it's for having the exact same situation when you press accidentally refresh button, open in a new tab, which is not the case right now. I think it can be more or less simple having an string variable with the IDs path concatenated by something like `/`, but indeed this is not a priority.

**Comment by petarlp** [comment]
Hello, i have same problems in odoo 14. 
When open in new tab some link from menu -> removes all menus
1.Main menu have id=336
http://localhost:8069/web#action=480&model=molekulite.clients&view_type=list&cids=1&menu_id=336
2. When open in new tab 
http://localhost:8069/web#action=484&cids=1&menu_id=343
menu_id is changed to submenu id (of which I have chosen )

Can you help me?
thanks
